src/prometheus-net.DotNetRuntime/StatsCollectors/ExceptionStatsCollector.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Prometheus.DotNetRuntime.EventSources;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+using System.Text;
+#if PROMV2
+using Prometheus.Advanced;
+#endif
+
+
+namespace Prometheus.DotNetRuntime.StatsCollectors
+{
+    public class ExceptionStatsCollector : IEventSourceStatsCollector
+    {
+        private const int EventIdExceptionThrown = 80;
+        private const string LabelReason = "exception";
+
+        internal Counter NumExceptions { get; private set; }
+        internal Counter ExceptionReasons { get; private set; }
+
+        public Guid EventSourceGuid => DotNetRuntimeEventSource.Id;
+
+        public EventKeywords Keywords => (EventKeywords)DotNetRuntimeEventSource.Keywords.Exception;
+        public EventLevel Level => EventLevel.Informational;
+
+        public void RegisterMetrics(MetricFactory metrics)
+        {
+            NumExceptions = metrics.CreateCounter(
```

Comment:
I think this counter is redundant- `dotnet_exception_reasons` could be `sum`ed to produce this value.